lesson-5/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample Project</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,700" rel="stylesheet">
+  </head>
+
+  <body>
+
+
+    <!-- PUT THE SAMPLE IMAGE AND YOUR CODE HERE -->
+<div Class=Box> 
```

Comment:
Indent this to be one step in from `body`, and in HTML markup, using all lowercase is more consistent